packages/kbn-securitysolution-grouping/src/components/group_selector/index.tsx

```diff
@@ -43,8 +43,17 @@ const GroupSelectorComponent = ({
     [groupsSelected]
   );
 
-  const panels: EuiContextMenuPanelDescriptor[] = useMemo(
-    () => [
+  const panels: EuiContextMenuPanelDescriptor[] = useMemo(() => {
+    const isOptionDisabled = (key?: string) => {
+      // Do not disable when maxGroupingLevels is 1 to allow toggling between groups
+      if (maxGroupingLevels === 1) {
```

Comment:
My main concern here is the different UX between `maxGroupingLevels = 1` and `maxGroupingLevels > 1`, while UI is almost the same. Maybe we should bring this with the designers who developed initial grouping UX if we haven't yet.
What I mean exactly:
here it will allow to go deep with the grouping and disable options when reach the level
<img width="256" alt="Screenshot 2023-11-02 at 11 56 07" src="https://github.com/elastic/kibana/assets/478762/9358c086-7c8c-407f-97c8-40795fa11c85">

while in our case it will work like a toggle. And the only differenciator between them is the number on top.
<img width="258" alt="Screenshot 2023-11-02 at 11 55 45" src="https://github.com/elastic/kibana/assets/478762/9dd6c698-d2d7-41f0-8056-9759fdc4529b">

 I totally agree that in case of max level 1 toggle makes much more sense, it still might be a bit confusing for users switching between Alerts and Findings. Maybe smth as simple as changing the text when max level 1 to smth like "Switch between grouping options" would do

Reply:
I agree, in the case of max level 1 would it be better to change the text to something like "Select grouping", I'll change it and bring it to discussion with the team who developed it when putting this PR to review.